### PR TITLE
Oracle build running on circleci

### PR DESCRIPTION
**Pull request by gsaslis**
**What this PR does / why we need it**:

Adds pipeline build against oracle database

**Which issue(s) this PR fixes** 

pipeline is not running against oracle in circleci

**Verification steps** 

`oracle_build` pipeline on circleci should be green

**Special notes for your reviewer**:

### CODE REVIEW

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -8,6 +8,19 @@ namespace :db do
     end
   end
 
+  desc 'Wait for database to boot, with `DB_BOOT_TIMEOUT` and `DB_BOOT_SLEEP_SECONDS` interval'
+  task :boot do
+    timeout = ENV['DB_BOOT_TIMEOUT'] || 300
```
Comment (hallelujah) [inline lib/tasks/db.rake]: Probably better:
timeout = ENV.fetch('DB_BOOT_TIMEOUT', 30).to_i

File: lib/tasks/db.rake [hunk 2]
```diff
@@ -8,6 +8,19 @@ namespace :db do
...
+    interval = ENV['DB_BOOT_SLEEP_SECONDS'] || 1
```
Comment (hallelujah) [inline lib/tasks/db.rake]: `interval = ENV.fetch('DB_BOOT_SLEEP_SECONDS', 1).to_i`

File: lib/tasks/db.rake [hunk 3]
```diff
@@ -8,6 +8,19 @@ namespace :db do
...
+
+    require 'system/database'
+    until System::Database.ready? || timeout == 0
```
Comment (hallelujah) [inline lib/tasks/db.rake]: `System::Database.ready? || timeout.negative?`

File: lib/tasks/db.rake [hunk 4]
```diff
@@ -8,6 +8,19 @@ namespace :db do
...
+      puts '.'
```
Comment (hallelujah) [inline lib/tasks/db.rake]: `print '.'`

File: lib/tasks/db.rake [hunk 5]
```diff
@@ -8,6 +8,19 @@ namespace :db do
...
+      sleep interval
+      timeout = timeout - 1
```
Comment (hallelujah) [inline lib/tasks/db.rake]: it should be `timeout = timeout - interval`

File: lib/tasks/ci/db.rake [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+  namespace :db do
+
+    desc 'Wait for database to boot, with `DB_BOOT_TIMEOUT` and `DB_BOOT_SLEEP_SECONDS` interval'
+    task :ready do
+      timeout = ENV.fetch('DB_BOOT_TIMEOUT', 300).to_i
+      interval = ENV.fetch('DB_BOOT_SLEEP_SECONDS', 1).to_i
+
+      if ENV['DB'] == 'oracle'
+        # allow some startup time for oracle to boot...  ¯\_(ツ)_/¯
```
Comment (didierofrivia) [inline lib/tasks/ci/db.rake]: 🤣

**Pull request merged**